siriuspy/siriuspy/clientconfigdb/types/as_diagnostics.py

```diff
@@ -1413,6 +1413,68 @@ def get_dict():
     ['SI-Glob:DI-TuneProc-V:FiltType-Sel', 0, 0],
     ]
 
+_fofbctrls = [
+    'IA-01RaBPM:BS-FOFBCtrl',
+    'IA-02RaBPM:BS-FOFBCtrl',
+    'IA-03RaBPM:BS-FOFBCtrl',
+    'IA-04RaBPM:BS-FOFBCtrl',
+    'IA-05RaBPM:BS-FOFBCtrl',
+    'IA-06RaBPM:BS-FOFBCtrl',
+    'IA-07RaBPM:BS-FOFBCtrl',
+    'IA-08RaBPM:BS-FOFBCtrl',
+    'IA-09RaBPM:BS-FOFBCtrl',
+    'IA-10RaBPM:BS-FOFBCtrl',
+    'IA-11RaBPM:BS-FOFBCtrl',
+    'IA-12RaBPM:BS-FOFBCtrl',
+    'IA-13RaBPM:BS-FOFBCtrl',
+    'IA-14RaBPM:BS-FOFBCtrl',
+    'IA-15RaBPM:BS-FOFBCtrl',
+    'IA-16RaBPM:BS-FOFBCtrl',
+    'IA-17RaBPM:BS-FOFBCtrl',
+    'IA-18RaBPM:BS-FOFBCtrl',
+    'IA-19RaBPM:BS-FOFBCtrl',
+    'IA-20RaBPM:BS-FOFBCtrl',
+]
+_fofb_propts = [
```

Comment:
@anacso17, are these triggers used only for diagnostics purposes? What are they used for? 

PS.: Maybe @ericonr could also answer these questions.

Reply:
We need to configure them so they don't affect the physical trigger buses, which are used mainly by the BPMs, at least for now.

We plan on using one of the lines for system identification; I don't know in which configuration that would fit, though.